### PR TITLE
Migrate tutorial doc client creation to point at new developers site

### PR DESCRIPTION
**Pull request by derek-globus**
Update the [sdk tutorial doc](https://globus-sdk-python.readthedocs.io/en/stable/tutorial.html) to use the new developers site.

----
:books: Documentation preview :books:: https://globus-sdk-python--802.org.readthedocs.build/en/802/

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -49,41 +49,38 @@ Step 1: Create a Client
...
-It's non-secure information and you can treat it as such.
+8. Copy the "Client UUID" from the page.
 
-In the rest of the tutorial we will assume in all code samples that it is
-available in the variable, ``CLIENT_ID``.
+   * This ID is can be thought of as your App's "username". It is non-secure information
+     and as such you can hardcode it into scripts, store it in a config file, or put it
+     into a database.
+
+In the rest of the tutorial we will assume in all code samples that it the Client UUID
```
Comment (sirosen) [inline docs/tutorial.rst:82]:
```suggestion
In the rest of the tutorial we will assume in all code samples that the Client UUID
```
Minor typo

File: docs/tutorial.rst [hunk 2]
```diff
@@ -49,41 +49,38 @@ Step 1: Create a Client
...
-You will need the Client ID from this screen.
-Feel free to think of this as your App's "username".
-You can hardcode it into scripts, store it in a config file, or even put it
-into a database.
-It's non-secure information and you can treat it as such.
+8. Copy the "Client UUID" from the page.
 
-In the rest of the tutorial we will assume in all code samples that it is
-available in the variable, ``CLIENT_ID``.
+   * This ID is can be thought of as your App's "username". It is non-secure information
```
Comment (sirosen) [inline docs/tutorial.rst:78]:
```suggestion
   * This ID can be thought of as your App's "username". It is non-secure information
```

File: docs/tutorial.rst [hunk 3]
```diff
@@ -49,41 +49,38 @@ Step 1: Create a Client
...
+     and as such you can hardcode it into scripts, store it in a config file, or put it
+     into a database.
```
Comment (sirosen) [inline docs/tutorial.rst:80]: Should we shorten this? I'm not sure I agree with my past self about the list of examples.

Maybe just "hardcode it in your scripts"?

File: docs/tutorial.rst [hunk 4]
```diff
@@ -49,41 +49,38 @@ Step 1: Create a Client
...
 
-    The **Native App** setting cannot be changed after a client is created.
-    If it is not selected during creation, you must create a replacement client.
+6. Give your App a name; this is what users will see when they are asked to
+   authorize your app.
 
-Expand the dropdown for the new app, and you should see an array of
-attributes of your client.
+7. Click "Register App". This will create your app and take you to a page
+   describing it.
```
Comment (sirosen) [inline docs/tutorial.rst:74]: We need to make sure it's a native app coming out of this process. I'll need to look at the new pages again to see how that's described, but I don't think confidential clients can use PKCE

**Review by sirosen — COMMENTED** [review]
In general I like the adjustments abd the flow of the updated doc. I have one thing I need to double check and a few minor typos to fix

File: docs/tutorial.rst [hunk 5]
```diff
@@ -49,41 +49,37 @@ Step 1: Create a Client
...
-Projects let you share the administrative burden of a collection of apps.
+1. Navigate to the `Developer Site <https://app.globus.org/settings/developers>`_
 
-In the "Add" menu for your Project, select "Add new app". To follow this
-tutorial, we will specify several values you should use.
+2. Select "Register a thick client or script that will be installed and run by users on
+   their devices."
 
-Enter the following pieces of information:
+3. Select "none of the above - create a new project" and click continue.
```
Comment (sirosen) [inline docs/tutorial.rst:60]: I just tested to confirm and the webapp skips this step if you have an account with no projects.
Not sure how exactly, but we should adjust. Maybe we make this a bullet point under (2)?

File: docs/tutorial.rst (hunk 4 above)
Comment (sirosen) [reply docs/tutorial.rst:74]: Gotcha; yes! I thought we were doing the flow in which you create and select a project, so the order of these steps got a bit mixed up in my head. I'm marking this part resolved.

File: docs/tutorial.rst (hunk 5 above)
Comment (derek-globus) [reply docs/tutorial.rst:60]: Hmm, yeah i can revise

Comment (derek-globus) [reply docs/tutorial.rst]: I restructured bullet points 3 & 4 into a share one.

**Review by sirosen — APPROVED** [review]
Awesome; thanks for iterating on this to refine the creation flow!

Also, 10/10 for removing a `warning` -- those are noisy and we should save them for when they're most needed.

**Pull request merged**